themes/base.theme.bash

```diff
@@ -620,6 +620,6 @@ function safe_append_prompt_command {
 }
 
 function _save-and-reload-history() {
-	local autosave=${1:-0}
-	[[ $autosave -eq 1 ]] && history -a && history -c && history -r
+	[[ ${1:-${autosave:-${HISTORY_AUTOSAVE:-0}}} -eq 1 ]] && local HISTCONTROL="${HISTCONTROL:-}${HISTCONTROL:+:}autoshare"
```

Comment:
I'd like to remove this function entirely, but I don't want to break anything in between now and then. That's why it's kinda shoehorned in there like that...

Reply:
@cornfeedhobo, let me know if you want me to specifically fix up this function to be less funky or if it's OK to leave it be since I plan to delete it in short order. (My PR to remove it is already up and marked draft pending this PR; see #1951.)